src/_handlers.ts

```diff
@@ -1,7 +1,7 @@
-import type * as express from 'express';
-import type { Options } from './types';
+import type { Request, Response, Options } from './types';
 import type * as httpProxy from 'http-proxy';
 import { getInstance } from './logger';
+import { getUrl } from './url';
```

Comment:
👀  as mentioned in issue discussions, we now have clarity that we cannot just use `req.url`--it's mutated from express. thus, from the `Request`, I have now _dropped_ `url: string`, as it's unsafe to access in the codebase. `getUrl` is a safe, express compatible accessor